consensus/constants.go

```diff
@@ -0,0 +1,11 @@
+package consensus
+
+// SubRoundBlockType defines the types of subround block type
+type SubRoundBlockType string
```

Comment:
Maybe we can keep subround, as a single word, without caps R everywhere in this file